Bitbucket-Pipelines/two-stage-pipeline-template/{{cookiecutter.outputDir}}/assume-role.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+echo $@
+ROLE=$1
+SESSION_NAME=$2
+PROFILE_NAME=$3
+
+unset AWS_SESSION_TOKEN
+
+aws configure --profile pipeline set aws_access_key_id $PIPELINE_USER_ACCESS_KEY_ID
+aws configure --profile pipeline set aws_secret_access_key $PIPELINE_USER_SECRET_ACCESS_KEY
```

Comment:
nit: Propose to make `pipeline` a bit more specific to reduce the chance for naming collision, like `sam-pipeline-user`.

Reply:
Yup, good idea. I'll make this change.